src/main/java/adapters/login_leaderboard/RegisterUserPresenter.java

```diff
@@ -1,10 +1,23 @@
 package adapters.login_leaderboard;
 
-/**
- * Calls on the registered_user and valid_password methods in RegisterUser to let the
- * user know if they have successfully registered. If they have an invalid password,
- * allow the user to re-enter a password.
- * Transforms the output data into a viewable format.
+import use_cases.login_leaderboard.IRegisterUserOutputBoundary;
+
+import javax.swing.*;
+
+
+/***
+ * The register use case presenter that aids in telling us whether
+ * the registry was successful or not.
  */
-public class RegisterUserPresenter {
+public class RegisterUserPresenter implements IRegisterUserOutputBoundary {
+
+    /***
+     * Displays a simple message pertaining to what the appropriate output is.
+     * @param u the response message, letting the user know if they've been
```

Comment:
just a suggestion, but maybe change u to a more descriptive variable and add a space before the parameter line, here and in other files